### ISSUE TITLE
DYNO_INTERFACE - how to specify copy constructor?

### ISSUE THREAD

**Issue by ArekPiekarz** [ldionne/dyno#40]
As in title, is it possible to do it now? Can dyno::CopyConstructible{} be used for that?
So far I wasn't able to write a compilable form of it.

**Comment by ldionne** [comment]
Good point; the `DYNO_INTERFACE` macro does not allow specifying special member functions at the moment. I'll need to think of a solution.
